icc-x-api/utils/efact-util.ts

```diff
@@ -333,11 +333,16 @@ function toInvoiceItem(
   const invoiceItem = new InvoiceItem({})
   invoiceItem.codeNomenclature = Number(invoicingCode.tarificationId!!.split("|")[1])
   invoiceItem.dateCode = dateEncode(toMoment(invoicingCode.dateCode!!)!!.toDate())
-  if (invoiceItem.codeNomenclature === 109594) {// Prétrajet de soins diabète
-    invoiceItem.endDateCode = dateEncode(
-      toMoment(invoicingCode.dateCode!!)!!
-        .endOf("month")
-        .toDate());
+  // Only applies to flatrate invoicing
```

Comment:
In this version, the 109954 case is not handled at all if no flarate. Is this intended?